cli/lib/kontena_cli.rb

```diff
@@ -87,6 +115,26 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    if log_target.respond_to?(:tty?) && log_target.tty?
+      logger = Logger.new(log_target)
+      logger.datetime_format = "%M%S.%2N "
```

Comment:
Some kind of "seconds since process start" would also be fine, but I doubt there's an easy `logger.datetime_format` for that.